nilearn/plotting/displays/_figures.py

```diff
@@ -103,3 +118,141 @@ def savefig(self, output_file=None):
         self._check_output_file(output_file=output_file)
         if self.figure is not None:
             self.figure.write_image(self.output_file)
+
+    def add_contours(self, roi_map, levels=None, labels=None, lines=None):
+        """
+        Draw boundaries around roi.
```

Comment:
Just for consistency

```suggestion
        """Draw boundaries around roi.
```